### ISSUE TITLE
[Bug] "Download completed" not entirely visible in landscape mode/"Open" button inaccessible

### ISSUE THREAD

**Issue by softvision-miralobontiu** [mozilla-mobile/fenix#6718]
### Steps to reproduce
1. Turn your device into landscape mode.
2. Download a file and wait for it to finish.
3. Observe the "Download completed" snackbar.

### Expected behavior
The snackbar is entirely visible and the "Open" button can be accessed.

### Actual behavior
The snackbar is not entirely visible, the "Open" button is not visible at all and cannot be accessed unless turning the device back into portrait mode.

### Device information
* Android device: all
* Fenix version: 11/20 Nightly build (#13240608)

![Screenshot_20191120-143727](https://user-images.githubusercontent.com/40694158/69239847-d4716f00-0ba3-11ea-96c0-866e621954c5.png)

**Comment by darkwing** [comment]
I cannot reproduce this in Fenix master:  
![Screenshot_20200211-211313](https://user-images.githubusercontent.com/46655/74279943-71463300-4ce1-11ea-8c28-a5d63a214a1a.png)

**Comment by abodea** [comment]
Verified as fixed on the latest **Nightly build from 2/12** with:

• Google Pixel 3a XL (Android 10)
• Google Pixel 3a (Android 9)
• Huawei Mate 20 Lite (Android 8.1.0)
• Samsung Galaxy S7 (Android 7.0.0)
• OnePlus A3 (Android 6.0.1)
• LG Nexus 4 (Android 5.1.1)
• Samsung Galaxy Tab A (Android 9).

Note that the `Download completed` was correctly displayed on all devices.
I also tried it in portrait mode.
